lib/elasticsearch-stream.js

```diff
@@ -41,7 +41,7 @@ class LibElasticsearchScrollStream extends Readable {
     this._total = typeof body.hits.total === 'object' ? body.hits.total.value : body.hits.total
 
     body.hits.hits.forEach(hit => {
-      let ref_results = hit.fields ? hit.fields : hit._source
+      let ref_results = hit.fields ? hit.fields : hit._source || hit
```

Comment:
```suggestion
      let ref_results = hit.fields ? hit.fields : hit._source || {}
```
I preferer this way here, and fix the related test

Reply:
👍  10d4564